src/animation/KeyframeTrack.d.ts

```diff
@@ -13,8 +13,8 @@ export class KeyframeTrack {
 	 */
 	constructor(
 		name: string,
-		times: any[],
-		values: any[],
+		times: Record<number, any>,
```

Comment:
In `BufferAttribute` the following approach is used:
https://github.com/mrdoob/three.js/blob/f3bfe7d3a7d58c8c41bdde335d61d256a06484c9/src/core/BufferAttribute.d.ts#L16
Could `ArrayLike` be used in `KeyframeTrack` for consistency, too?